### ISSUE TITLE
work on figure 2 -- timeline of diversity

### ISSUE THREAD

**Issue by lizzieinvancouver** [lizzieinvancouver/asevdiversitydup#3]
See [figure here](https://github.com/lizzieinvancouver/asevdiversitydup/blob/main/figures/scan21Apr2023_historyfig.png)

This one I am pretty hopeful we'll include -- just try to re-create it as best you can. Some background reading on the ancient history [in this new paper](https://www.science.org/doi/10.1126/science.add8655)

**Comment by christophe-rd** [comment]
I started a draft today, but haven't included all the information yet. I pushed it in the repo in case you wanted to take a quick look already!

**Comment by lizzieinvancouver** [comment]
@christophe-rd 

For figure timeline diversity, it looks good! I suggest making the periods smaller or maybe putting them within boxes as is often done (see Fig 1 [here](https://www.pnas.org/doi/10.1073/pnas.1809600115) for an example) and use a sans serif font for figures; I like Helvetica).

**Comment by christophe-rd** [comment]
I updated the periods and added the missing information. I pushed two versions, one simple and one with two arrows. The first is for grapevine history and the other is related to human migration. I just put the information from [the article you previously provided](https://www.science.org/doi/10.1126/science.add8655) for the moment. Let me know which version you like the better, and please tell me which changes I need to make!

**Comment by lizzieinvancouver** [comment]
@christophe-rd This looks amazing! I don't have any edits to suggest now. Thank you!

**Comment by lizzieinvancouver** [comment]
@christophe-rd One thing -- since we're using a version control program (git) you should rarely need things with v1 etc. -- instead just update the file (the old one is preserved). So I suggest you have only one .ai and one .png of each figure. You can delete the v1 parts and other versions (they are easy to find).

**Comment by christophe-rd** [comment]
Perfect that sounds good! Sorry for the delay in my response. 
For Figure 2, since I created two recent versions, I kept them both and named them 2.1 and 2.2. I deleted the V1 from Figure 1!

**Comment by lizzieinvancouver** [comment]
@christophe-rd Sounds good! The figures look great right now. I am working on a new draft so will be in touch in a couple days when I have more time to see where everything is at.

**Comment by lizzieinvancouver** [comment]
Could we make a few tweaks to this?

- [ ] At 1620 add `Listan Prieto' introduced to Baja California 
- [ ] Add a line for 1769-1833 for 'California missions'
- [ ] If possible, increase the font size throughout a little, and more for the grapevine history section
- [ ] Make the jagged clouds for diversity lost and gained to be filled in with the color and make the color very faint
- [ ] Italicize Vitis wherever we use it.

**Comment by christophe-rd** [comment]
@lizzieinvancouver yes! I will update the figure this week and will let you know when these adjustments are made!

**Comment by lizzieinvancouver** [comment]
@christophe-rd Great -- thank you!

**Comment by christophe-rd** [comment]
Hi @lizzieinvancouver! I asked my girlfriend yesterday to help me with Illustrator (she works with it every day). She suggested a few edits to make the visual more appealing and easier to read. We need to finalize it, but I wanted to run this by you before we finish it. I will change the diversity loss and gain symbol. Could you please let me know if you like the direction the figure is taking? Of course, it misses a lot of information and is probably only halfway done!
[Here's the .ai ](https://github.com/lizzieinvancouver/asevdiversitydup/blob/2038b6f91fd9b33d04bce6dd25c18a2457b65581/figures/figure_2.2.ai)
[Here's the .pdf.](https://github.com/lizzieinvancouver/asevdiversitydup/blob/2038b6f91fd9b33d04bce6dd25c18a2457b65581/figures/figure_2.2.pdf)

**Comment by lizzieinvancouver** [comment]
I love it! The diversity gains go with the new world `discovery' and the losses go with domestication and Phylloxera, so we should make sure that is clear ... but overall I really like the changes. My thanks to your girlfriend!

**Comment by christophe-rd** [comment]
Perfect! I'll make those adjustments and will let you know once we are close to being done!

**Comment by lizzieinvancouver** [comment]
@christophe-rd Great! Once we have an updated draft I will send all the new figures to colleagues to make sure they're happy.

**Comment by christophe-rd** [comment]
> The diversity gains go with the new world `discovery' and the losses go with domestication and Phylloxera, so we should make sure that is clear ... but overall I really like the changes. My thanks to your girlfriend!

[Here's the "final version".](https://github.com/lizzieinvancouver/asevdiversitydup/blob/c2823217f9998099ee91a5950fc56e48a18d7808/figures/figure_2.2.pdf) I am uncertain if the diversity gains and losses bubbles were going precisely to those events you mentioned or if they were more "general". Let me know what you think!

I know this figure has changed a lot, but please feel free to let me know if there is anything you would like me to change. Also, do you think we should ref this [article](https://www.science.org/doi/10.1126/science.add8655) since I was "inspired" a lot by their timeline figure?

**Comment by lizzieinvancouver** [comment]
@christophe-rd Re:

> Before common era (BCE) versus 11,000 years ago. Do you think I should change it to avoid confusion?

I find years before present much easier, so let's change it if that's easy (otherwise, the legend already does the conversion). I also referenced Dong and the source I used for the missions in the caption for dates. The general timeline design of the figure is fairly common so I don't think we should give credit to Dong et al for that. 

I made some small edits to the current figure (so please git pull). I wanted to make the 'selective sweeps' arrow fade out to slightly transparent at the end (as I cannot tell when those ended, and I suspect the jury is out ... if you think Dong et al. 2023 makes it clear though, let me know). Can you find a way to make the line+arrow go from black to transparent?

**Comment by christophe-rd** [comment]
>I find years before present much easier, so let's change it if that's easy (otherwise, the legend already does the conversion). I also referenced Dong and the source I used for the missions in the caption for dates. The general timeline design of the figure is fairly common so I don't think we should give credit to Dong et al for that.

Sounds good! I got a little confused when it got to the year 1500. Are years before the present only relevant before the common era starts? E.g. on the year 1500, should it be "500 years before present" or just 1500? [See attached](https://github.com/lizzieinvancouver/asevdiversitydup/blob/2324086fb56348723114b070929d4c7ca4d6cd4b/figures/figure_2.2.png)

>I made some small edits to the current figure (so please git pull). I wanted to make the 'selective sweeps' arrow fade out to slightly transparent at the end (as I cannot tell when those ended, and I suspect the jury is out ... if you think Dong et al. 2023 makes it clear though, let me know). Can you find a way to make the line+arrow go from black to transparent?

Perfect! I found a way and the new version is pushed back.

**Comment by lizzieinvancouver** [comment]
> I got a little confused when it got to the year 1500. Are years before the present only relevant before the common era starts? E.g. on the year 1500, should it be "500 years before present" or just 1500?

Yeah, this is more confusing than I recalled! And the convention for 'present' seems to depend on discipline. If you git pull I made a version with both dates -- I think it works okay ... and probably as good as possible given the different conventions across fields. We can see what collaborators say.

**Comment by christophe-rd** [comment]
> Yeah, this is more confusing than I recall! And the convention for 'present' seems to depend on discipline. If you git pull I made a version with both dates -- I think it works okay ... and probably as good as possible given the different conventions across fields. We can see what collaborators say.

Perfect, thanks! I updated 2 quick versions of the ellipses that Thierry suggested. In addition with the changes he suggested:
[Broad](https://github.com/lizzieinvancouver/asevdiversitydup/blob/ae755088886a5379a4e6efe90bf0d196e7958054/figures/figure_2.2_large.png)
[2 elipses](https://github.com/lizzieinvancouver/asevdiversitydup/blob/ae755088886a5379a4e6efe90bf0d196e7958054/figures/figure_2.2_2elipses.png)

Could you please let me know your opinion on this?

**Comment by christophe-rd** [comment]
I also changed ENTAV with Clonal selection (ENTAV creation).

**Comment by lizzieinvancouver** [comment]
@christophe-rd They look good! Thanks for the quick work on this. I think Thierry prefers the broad one and I like it too so let's go with that. I don't understand his whole email though so will follow up ....

**Comment by lizzieinvancouver** [comment]
@christophe-rd I didn't see the ENTAV change so adding it now.

**Comment by christophe-rd** [comment]
> @christophe-rd I didn't see the ENTAV change so adding it now.

My mistake! I changed it on the .ai, but didn't update the PNGs after!

**Comment by lizzieinvancouver** [comment]
@christophe-rd No worries!
